cmd/nerdctl/volume_rm.go

```diff
@@ -39,14 +43,52 @@ func newVolumeRmCommand() *cobra.Command {
 }
 
 func volumeRmAction(cmd *cobra.Command, args []string) error {
+	client, ctx, cancel, err := newClient(cmd)
+	if err != nil {
+		return err
+	}
+	defer cancel()
+	containers, err := client.Containers(ctx)
+	if err != nil {
+		return err
+	}
+	var mount *specs.Spec
+	var removednames []string
```

Comment:
`var removedNames` would be better?

Reply:
I used it for volumes that have to be deleted.